scripts/scripts.js

```diff
@@ -535,14 +543,24 @@ window.store = new (class {
    */
   query(block) {
     const id = getBlockId(block);
-    const query = this.getQuery(block);
+    let query = this.getQuery(block);
 
     if (!query) {
-      console.warn(`Query missing for "${block.dataset.blockName}" with id "${id}"`);
-      return;
+      this.initQueries();
```

Comment:
@DriftingSands  You might want to also add a comment explaining why initQueries is called again to support blocks that were loaded after the initial block loading phase.